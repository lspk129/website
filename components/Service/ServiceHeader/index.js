import styled from 'styled-components';

const ServiceHeader = styled.h2`
  text-align: center;
  padding: 60px;
  margin: 0;
`;

export default ServiceHeader;
