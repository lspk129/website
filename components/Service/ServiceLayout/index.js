import styled from 'styled-components';

const ServiceLayout = styled.div`
  text-align: center;
  display: grid;
  grid-template-rows: 1fr 3fr;
`;

export default ServiceLayout;
