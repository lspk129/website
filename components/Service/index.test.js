import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

import ServiceSection from './';
import ServiceLayout from './ServiceLayout';
import ServiceHeader from './ServiceHeader';

describe('ServiceSection', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<ServiceSection />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders ServiceLayout component', () => {
    const wrapper = shallow(<ServiceSection />);
    expect(wrapper).toBePresent();
    expect(wrapper.find(ServiceLayout)).toBePresent();
  });

  it('renders ServiceHeader component', () => {
    const wrapper = shallow(<ServiceSection />);
    expect(wrapper).toBePresent();
    expect(wrapper.find(ServiceHeader)).toBePresent();
  });

  it('renders text "What we do"', () => {
    const wrapper = mount(<ServiceSection />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveText('What we do');
  });
});
