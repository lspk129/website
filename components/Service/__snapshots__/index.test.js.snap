// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Service renders correctly 1`] = `
<div
  className="sc-bdVaJa hJyKNr"
>
  <h2
    className="sc-bwzfXH fSlUUc"
  >
    What we do
  </h2>
  <div
    className="sc-htpNat jdAldW"
  >
    <div
      className="sc-bxivhb doscWV"
    >
      <i
        className="material-icons"
      >
        desktop_mac
      </i>
      <h3>
        Web development
      </h3>
      <p>
        We help you build fast, functional, performance oriented websites and web apps
      </p>
    </div>
    <div
      className="sc-bxivhb doscWV"
    >
      <i
        className="material-icons"
      >
        color_lens
      </i>
      <h3>
        Web design
      </h3>
      <p>
        We are creating unique and adaptive designs to engage customers
      </p>
    </div>
  </div>
</div>
`;
